includes/login.php

```diff
@@ -3,14 +3,22 @@
   $email    = $_POST['email'];

```

Comment:
There is probably no need to create an alias for this value since it is used immediately and not changed (?) .